recipes/clhpp/LICENSE.txt

```diff
@@ -0,0 +1,25 @@
+Copyright (c) 2008-2015 The Khronos Group Inc.
```

Comment:
FYI the license should be in the next release. It is already in `master`.

xref: https://github.com/KhronosGroup/OpenCL-CLHPP/pull/21